frontend/packages/console-shared/src/components/dashboard/status-card/HealthItem.tsx

```diff
@@ -5,28 +5,28 @@ import { DashboardCardPopupLink } from '../dashboard-card/DashboardCardLink';
 import { HealthState, healthStateMapping, healthStateMessage } from './states';
 import { SecondaryStatus } from '../../status';
 
-const HealthItemIcon: React.FC<HealthItemIconProps> = ({ state }) => (
-  <div className="co-dashboard-icon">
+const HealthItemIcon: React.FC<HealthItemIconProps> = ({ state, dataTest }) => (
+  <div data-test={dataTest} className="co-dashboard-icon">
     {(healthStateMapping[state] || healthStateMapping[HealthState.UNKNOWN]).icon}
   </div>
 );
 
 const HealthItem: React.FC<HealthItemProps> = React.memo(
-  ({ className, state, title, details, popupTitle, noIcon = false, children }) => {
+  ({ className, state, title, details, popupTitle, noIcon = false, dataTest, children }) => {
```

Comment:
I dont see any code calling `HealthItem` with this `dataTest` prop .. ?

Reply:
Right, fixed!